src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/interior_data_cache_impl.cc

```diff
@@ -88,6 +94,22 @@ void InteriorDataCacheImpl::ClearCache() {
 void InteriorDataCacheImpl::ResetRequestCountOnTimer() {
   LOG4CXX_AUTO_TRACE(logger_);
   amount_of_request_in_this_time_frame_ = 0;
+  std::this_thread::sleep_for(
+      std::chrono::seconds(time_frame_of_allowed_requests_));
+}
+
+void InteriorDataCacheImpl::StartRequestResetTimer(
+    const uint32_t time_frame_of_allowed_requests) {
+  if (!reset_request_count_timer_.is_running()) {
+    reset_request_count_timer_.Stop();
+  }
+
+  time_frame_of_allowed_requests_ = time_frame_of_allowed_requests;
+  const uint32_t translate_msec_to_sec = 1000;
```

Comment:
@ZhdanovP you can fetch this constants from date time

Reply:
@LuxoftAKutsan done ff6979d4bdcbd3dca9dd6a79d9a02e83c4c40788